twitter4j-core/src/main/java/twitter4j/User.java

```diff
@@ -46,6 +46,13 @@
     String getScreenName();
 
     /**
+     * Returns true if it's following the logged in user
+     *
+     * @return the following of the user
```

Comment:
Suggest replacing "the following of the user" with "true if the authenticated user is following this user"